deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -133,7 +137,7 @@ def __init__(self, manager_address, cypher="aes", os=None, inventory_sample=None
             "fim": {"status": "enabled", "eps": self.fim_eps},
             "fim_integrity": {"status": "disabled", "eps": self.fim_integrity_eps},
             "syscollector": {"status": "disabled", "frequency": 60.0, "eps": self.syscollector_eps},
-            "rootcheck": {"status": "disabled", "frequency": 60.0, "eps": self.rootcheck_eps},
+            "rootcheck": {"status": "disabled", "frequency": rootcheck_frequency, "eps": self.rootcheck_eps},
```

Comment:
Follow the used format for the rest of the Agent attributes

```suggestion
            "rootcheck": {"status": "disabled", "frequency": self.rootcheck_frequency, "eps": self.rootcheck_eps},
```